python/publish_telemetry.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import asyncio
+from inspect import ArgSpec
+import time
+import argparse
+from tcpClient import TopicType, TcpClient
+
+tcp_client_cmd = TcpClient()
+tcp_client_tel = TcpClient() 
+
+localhost = "127.0.0.1"
+
+# Set the parser
+parser = argparse.ArgumentParser()
+parser.add_argument("port_cmd", type=int, metavar="pcmd", help="set the command port")
+parser.add_argument("port_tel", type=int, metavar="ptel", help="set the telemetry port")
+parser.add_argument("--ncmd", type=int, metavar="N", default=10, help="set the number of commands to be sent (default 10)")
+#parser.add_argument("--frec", type=int, metavar="F", default=10, help="set the frecuency in Hz of the telemetry to be published (default 10 Hz)")
+#parser.add_argument("--dur", type=int, metavar="D", default=2, help="set the duration in seconds of the telemetry to be published (default 2 seconds)")
```

Comment:
You can remove the commented code if you do not need them anymore. This applies to all commented code in this script. In addition, you may need to add a new line at the end of file because of this:
![image](https://github.com/lsst-ts/ts_labview_tcp_json/assets/11474902/b2eed685-3f84-4e86-9493-324341813210)

Reply:
Done. Thanks.